sormas-ui/src/test/java/de/symeda/sormas/ui/event/eventparticipantimporter/EventParticipantImporterTest.java

```diff
@@ -177,7 +177,7 @@ public void testImportEventParticipantSimilarityPick()
 		assertEquals(1, eventParticipantFacade.count(new EventParticipantCriteria().withEvent(eventRef)));
 		assertEquals(person.getUuid(), importedEventParticipant.getPersonUuid());
 		assertEquals(person.getFirstName(), importedPerson.getFirstName());
-		assertEquals(person.getLastName(), importedPerson.getLastName());
+		assertEquals("Heinze", importedPerson.getLastName());
```

Comment:
Using an explicit string here might not be a good idea because it will make the test fail when someone changes the test .csv file. What's the reason for this change?